APMrover2/Parameters.cpp

```diff
@@ -125,7 +125,7 @@ const AP_Param::Info Rover::var_info[] = {
     // @Param: SPEED_TURN_GAIN
     // @DisplayName: Target speed reduction while turning
     // @Description: The percentage to reduce the throttle while turning. If this is 100% then the target speed is not reduced while turning. If this is 50% then the target speed is reduced in proportion to the turn rate, with a reduction of 50% when the steering is maximally deflected.
-    // @Units: percent
```

Comment:
I actually think "percent" or "percentage" is a better unit then %. Just my $0.02